src/styles/pages/sitesAndApplications.module.scss

```diff
@@ -26,25 +26,125 @@
 }
 
 .widgetContainer {
-  border: 1px solid color('base-light');
+  border: 1px solid var(--widget-border);
   padding: 3px;
   border-radius: 4px;
   box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.15);
 
   &.selectMode {
-    border: 4px solid $theme-spacebase-base;
+    border: 4px solid var(--widget-selected-border);
     padding: 0px;
 
     .widgetToolbar {
-      background-color: $theme-spacebase-base;
+      background-color: var(--widget-selected-border);
     }
   }
 
   .widgets {
     margin: units(4) units(4) units(6);
   }
+
+  :global(.sitesAndAppsCollection) {
+    background-color: var(--widget-sites-apps-bg);
+    border-color: var(--bookmark-sites-apps-border);
+
+    :global(.sitesAndAppsBookmark) {
+      background-color: var(--bookmark-sites-apps-bg);
+      border-color: var(--bookmark-sites-apps-border);
+    }
+  }
 }
 
+[data-color-theme='dark'] {
+  .widgetContainer {
+    border: none;
```

Comment:
I think there should be a border here? In Figma there is a border with what looks like a transparent bg for the container. Apologies if this was discussed elsewhere and I missed it!

Reply:
thanks -- i'll take a look!

Reply:
fixed 🌠